app/views/content/a-day-in-the-life-of-a-teacher.md

```diff
@@ -17,79 +18,145 @@
     - everyday
     - timetable
   quote:
-    q-sarah-primary:
-      text: "I like to get in early so I can get organised — in teaching, preparation is everything."
-      name: "Sarah, primary school teacher"
-      classes: quote--indent
-    q-sarah-geography-1:
-      text: "I have four free periods a week, which I use as my own time."
-      name: "Sarah, geography teacher"
-      classes: quote--indent
-    q-gavin:
-      text: "Teaching and learning does not stop with the bell at the end of the school day."
+    q-gavin-ict:
+      text: "Getting involved with extracurricular activities can be extremely rewarding."
       name: "Gavin, assistant headteacher"
       classes: quote--indent
-    q-sarah-geography-2:
+    q-jane:
+      text: "It’s all about the relationships you build with pupils. If they know they’ll ‘get away’ with disruptive learning, they will disrupt learning!"
+      name: "Jane, teaching adviser"
+      classes: quote--indent
+    q-sarah-geography:
       text: "I sometimes do a couple of hours at the weekend but I try to keep my weekends free to make sure I have some real downtime."
       name: "Sarah, geography teacher"
       classes: quote--indent
+    q-helen-food:
+      text: "It's not an easy career. There will be challenges along the way but persevere and the rewards are amazing. It's my 35th year and every day is still different. Do not expect too much of yourself too soon - 1 day at a time but you will make a difference and you will be remembered."
```

Comment:
```suggestion
      text: "It's not an easy career. There will be challenges along the way but persevere and the rewards are amazing. It's my 35th year and every day is still different. Do not expect too much of yourself too soon. You'll make a difference and be remembered."
```
I didn't think this quote flowed quite right - I've tried to make it a bit smoother but my suggestion might not be the right solution.